pkg/flag/global_flags.go

```diff
@@ -144,6 +145,8 @@ func (f *GlobalFlagGroup) ToOptions() (GlobalOptions, error) {
 	// Keep TRIVY_NON_SSL for backward compatibility
 	insecure := f.Insecure.Value() || os.Getenv("TRIVY_NON_SSL") != ""
 
+	log.Debug("Cache dir", log.String("dir", f.CacheDir.Value()))
```

Comment:
i am not sure that we need to add this here.
We show cache dir for `server` mode: https://github.com/aquasecurity/trivy/blob/5751870b0fef5cf624a94cbf6edbd0cc238acb46/pkg/commands/server/run.go#L27
(perhaps it make sense to move this to https://github.com/aquasecurity/trivy/blob/5751870b0fef5cf624a94cbf6edbd0cc238acb46/pkg/cache/remote.go#L31-L47)

Also we show this log for redis cache (cache directory information when using `redis` can be confusing for users).

But we need to add this log to https://github.com/aquasecurity/trivy/blob/5751870b0fef5cf624a94cbf6edbd0cc238acb46/pkg/cache/fs.go#L22

Reply:
>Also we show this log for redis cache (cache directory information when using redis can be confusing for users).

`--cache-dir` is effective more than the scan cache. It also affects the vulnerability DB and the Java DB.

Reply:
>We show cache dir for server mode:

Deleted https://github.com/aquasecurity/trivy/pull/7024/commits/1ab584ef6614045347be37423d07fafdb16e6093

Reply:
> --cache-dir is effective more than the scan cache. It also affects the vulnerability DB and the Java DB.

hm... you are right.
Let's update redis log then (e.g. `Redis scan cache`):
https://github.com/aquasecurity/trivy/blob/1ab584ef6614045347be37423d07fafdb16e6093/pkg/cache/redis.go#L91

Reply:
BTW, I'm not sure scan cache is a good name. I tentatively named it "scan cache" because we needed a different name to distinguish between a general cache, including DB, and a specific cache for storing analysis results, but there are other candidates, such as "layer cache" or "analysis cache". What do you think?

Reply:
I don't like `layer cache`. User may think that we store layers (I mean files from layers), but we only store layer IDs.

`analysis cache` looks good.

Reply:
>I don't like layer cache. User may think that we store layers (I mean files from layers), but we only store layer IDs.

Right. That's why I didn't choose it.

While "analysis cache" describes the internal logic well, the "analyzer" concept is not exported to end users. I'm not sure people understand "analysis cache". Anyway, I'll change it to Redis scan cache for now. I'll open another issue to discuss the naming.

Reply:
Fixed https://github.com/aquasecurity/trivy/pull/7024/commits/2868b08d02d7da7039c926237065faca37780037

Reply:
>  the "analyzer" concept is not exported to end users. I'm not sure people understand "analysis cache"

We, as maintainers, constantly work with `analyzers` and are accustomed to this term, so perhaps `scan cache` does not seem to be best choice for us, but perhaps it will be clearer for users

Reply:
@DmitriyLewen Please feel free to leave any comments.
https://github.com/aquasecurity/trivy/discussions/7034